Submodels/MTSOL_call.py

```
@@ -1,20 +1,613 @@
+        self.process.stdin.flush()
+
+        # Write inlet Mach number
+        self.process.stdin.write(f"M {self.operating_conditions["Inlet_Mach"]} \n")
+        self.process.stdin.flush()
+
+        # Set critical amplification factor to N=9 rather than the default N=7
+        self.process.stdin.write("N 9\n")
+        self.process.stdin.flush()
+
+        # Set the Reynolds number to 0 to ensure an inviscid solve is performed initially
+        self.process.stdin.write(f"R 0 \n")
```

Ruff (0.8.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)